Pipfile

```diff
@@ -12,6 +12,8 @@ pymdown-extensions = "*"
 mdx-include = "*"
 yamllint = "*"
 mkdocs-markdownextradata-plugin = "*"
+mkdocs-mermaid-plugin = {git = "https://github.com/PeterJCLaw/mkdocs-mermaid-plugin",ref = "python-3"}
+beautifulsoup4 = "*"
```

Comment:
Should this not be added as a dependency of the above, as it's not actually used in this application

Reply:
Quite possibly.

Reply:
From what I recall, this was added like this because it _wasn't_ being added as a transient dependency (due to bugs in the upstream; though those are somewhat fixed by using my fork).

Reply:
From what I recall, this was added like this because it _wasn't_ being added as a transient dependency (due to bugs in the upstream; though those are somewhat fixed by using my fork).

Reply:
The upstream has now merged my fixes, so this can now be updated. It's still not updated on PyPI, but we'll see if that happens.

Reply:
376c7ecf4e162069c96faeead8b8d68830b17117 improves this, but still not on PyPI.

Reply:
7defe3f fixes the inclusion of `beautifulsoup4`.